src/controllers/helper.js

```diff
@@ -0,0 +1,152 @@
+const { Book, Reader, Author, Genre } = require("../models");
+
+const getModel = (model) => {
+  switch (model) {
+    case Book:
+      return "book";
+      break;
+    case Reader:
+      return "reader";
+      break;
+    case Author:
+      return "author";
+      break;
+    case Genre:
+      return "genre";
+      break;
+  }
+};
+
+const hidePassword = (data) => {
+  const password = data.password;
+  const split = password.split("");
+  const starredArray = split.map((char) => {
+    return "*";
+  });
+  const starred = starredArray.join("");
+
+  data.password = starred;
+  return data;
+};
```

Comment:
This could be made much simpler:
```suggestion
const hidePassword = (data) => {
  const starredPassword = new Array(data.password.length + 1).join('*');
  data.password = starredPassword;
  return data;
};
```